scipy/stats/_hypotests.py

```diff
@@ -1433,3 +1435,288 @@ def cramervonmises_2samp(x, y, method='auto'):
             p = max(0, 1. - _cdf_cvm_inf(tn))
 
     return CramerVonMisesResult(statistic=t, pvalue=p)
+
+
+class Tukey_HSDResult:
+    """Result of `scipy.stats.tukey_hsd`.
+
+    Attributes
+    ----------
+    statistic : float ndarray
+        The computed statistic of the test for each comparison. The
+        ``(i, j)`` index is the statistic for the ith - jth group comparison.
+    pvalue : float ndarray
+        The associated p-value from the studentized range distribution. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
+
+    Methods
+    -------
+    confidence_interval :
+        Compute the confidence interval for the significance level
```

Comment:
Please search for "significance level" and change to "confidence level".

Reply:
Sorry, I got a little confused between confidence interval and confidence level.